plugins/module_utils/database.py

```diff
@@ -29,6 +29,21 @@
 import re
 
 
+# Input patterns for is_input_dangerous function:
+#
+# 1. '"' in string and '--' in string or
+# "'" in string and '--' in string
+PATTERN_1 = re.compile(r'(\'|\").*--')
+
+# 2. union \ intersect \ except + select
+PATTERN_2 = re.compile(r'(union|UNION|intersect|INTERSECT|'
+                       r'except|EXCEPT).*(select|SELECT)')
+
+# 3. ';' and any KEY_WORDS
+PATTERN_3 = re.compile(r';.*(select|SELECT|update|UPDATE|insert|INSERT|'
+                       r'delete|DELETE|drop|DROP|truncate|TRUNCATE|alter|ALTER)')
```

Comment:
I might be wrong but doesn't PostgreSQL accept mixed case for keywords e.g. `Select`, would it be better to use a case insensitive regex? Maybe something like
```suggestion
PATTERN_3 = re.compile(r';.*(SELECT|UPDATE|INSERT|DELETE|DROP|TRUNCATE|ALTER)', re.IGNORECASE)
```

Reply:
oh, it's a bit strange but it allows to use ```SeLect```:) Thanks!